python-main.js

```diff
@@ -903,6 +919,7 @@ function web_editor(config) {
                     inputFile.value = '';
                     vex.close();
                     EDITOR.focus();
+                    invalidExt(ext);
```

Comment:
Maybe we can call `invalidExt` in an else statement where we are checking the extensions in line 917.

Reply:
Do you want the Load/Save modal to close if the user enters an invalid file or do you just want another modal to appear in front of it?

Reply:
A modal on top of a modal would be a bit confusing, better to close the load/save modal and then show the error modal.